pkgs/by-name/pr/presenterm/package.nix

```
@@ -6,20 +6,20 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "presenterm";
-  version = "0.4.1";
+  version = "0.5.0";
 
   src = fetchFromGitHub {
     owner = "mfontanini";
     repo = "presenterm";
     rev = "refs/tags/v${version}";
-    hash = "sha256-OHp/qbuaZ7uVydKGnSiBR5KQGdf8rWQQWRHrka+PI1M=";
+    hash = "sha256-WdyGpJCCBez7CRrkgJHb5BL3UMwgPiCoYU2xVS2x+cU=";
```

```suggestion
    hash = "sha256-VAcK02dbtuTGn+lPu1vb/wAkroHuHqsU2KYHBiP2Org=";
```

Probably also need to update `cargoHash`.

---

Done ! `cargoHash` is fine